commit 7b1bcf2ccbe51ee6f5f5e32a1b6d0909542637f3
Author: Serge Schneider <serge@raspberrypi.org>
Date:   Tue Oct 2 11:14:15 2018 +0100

    drivers: thermal: step_wise: avoid throttling at hysteresis temperature after dropping below it
    
    Signed-off-by: Serge Schneider <serge@raspberrypi.org>

diff --git a/drivers/thermal/step_wise.c b/drivers/thermal/step_wise.c
index cf07e2269291..da695d8f2939 100644
--- a/drivers/thermal/step_wise.c
+++ b/drivers/thermal/step_wise.c
@@ -167,7 +167,7 @@ static void thermal_zone_trip_update(struct thermal_zone_device *tz, int trip)
 		 */
 		if (tz->temperature >= trip_temp ||
 		   (tz->temperature >= hyst_temp &&
-		   old_target != THERMAL_NO_TARGET)) {
+		   old_target == instance->upper)) {
 			throttle = true;
 			trace_thermal_zone_trip(tz, trip, trip_type);
 		}
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1643_7b1bcf2ccbe51ee6f5f5e32a1b6d0909542637f3.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1643_7b1bcf2ccbe51ee6f5f5e32a1b6d0909542637f3.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
